plugin/completion/lib_complete.py

```diff
@@ -301,7 +304,27 @@ def _cindex_for_version(version):
         return None
 
     @staticmethod
-    def _parse_completions(complete_results):
+    def _is_valid_result(completion_result, excluded_kinds):
+        """Check if completion is valid. Remove excluded types and unaccessible members
+
+        Args:
+            completion_result (): completion result from libclang
+            excluded_kinds (list): list of CursorKind types that shouldn't be added to completion list
+
+        Returns:
+            boolean: True if completion should be added to completion list
+        """
+        if str(completion_result.string.availability) != "Available":
+            return False
+        try:
+            if completion_result.kind in excluded_kinds:
+                return False
+        except Exception as e:
```

Comment:
Do we know which exception are we expecting? I think it is better to catch only those that can occur.